test/library/gui/widgets.TextArea.lua

```diff
@@ -0,0 +1,3324 @@
+local gui = require('gui')
+local widgets = require('gui.widgets')
+
+config.target = 'core'
+
+local df_major_version = tonumber(dfhack.getCompiledDFVersion():match('%d+'))
```

Comment:
now that 50.14 has been released (which supports the new cursor navigation keybindings), you can take this out and run the tests unconditionally